packages/logstash/data_stream/node_cel/fields/agent.yml

```
@@ -0,0 +1,141 @@
+- name: cloud
+  title: Cloud
+  group: 2
+  description: Fields related to the cloud or infrastructure the events are coming from.
+  footnote: "Examples: If Metricbeat is running on an EC2 host and fetches data from its host, the cloud info contains the data about this machine. If Metricbeat runs on a remote machine outside the cloud and fetches data from a service running in the cloud, the field contains cloud data from the machine the service is running on."
+  type: group
+  fields:
+    - name: account.id
+      type: keyword
+      ignore_above: 1024
+      description: "The cloud account or organization id used to identify different entities in a multi-tenant environment.\nExamples: AWS account id, Google Cloud ORG Id, or other unique identifier."
```

A lot of these fields exist in ECS and should use `external: ecs`.

```suggestion
    - name: account.id
      external: ecs
```

I've been working on a tool to fix this automatically. This command will edit the file for you. (I would only use it on a clean workspace so you can see review its edits.)

`go run github.com/andrewkroh/fydler@main -fix packages/logstash/data_stream/node_cel/fields/*yml`